jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/FrameFlusher.java

```diff
@@ -126,6 +150,14 @@ public boolean enqueue(Frame frame, Callback callback, boolean batch)
             }
         }
 
+        if (failedEntries != null)
+        {
+            for (Entry e : failedEntries)
+            {
+                notifyCallbackFailure(e.callback, new WebSocketException(CloseStatus.codeString(closeStatus.getCode())));
```

Comment:
The exception is a constant, don't create it multiple times.

Reply:
Please add some descriptive text before `CloseStatus.codeString()`.

Reply:
remember that the stack might not be constant!